primer/index.bs

```diff
@@ -98,7 +98,7 @@ Several actors are at play in our example Solid-OIDC authentication flows:
   <dd>Decent Photos is a Solid compliant application and has a URI of its own,
  which resolves to a Client ID Document. An RP's Client ID Document
  contains information identifying them as a registered OAuth 2.0 client
- application. Decent Photo's URI is `https://decentphotos.example/webid#this`</dd>
+ application. Decent Photo's URI is `https://decentphotos.example/client_id`</dd>
```

Comment:
I've changed this value everywhere to be `https://decentphotos.example/client_id` — I don't think the hash segment would be meaningful here / I've never seen it used. At a bare minimum, we'd want `https://decentphotos.example/client_id#this`